assessments/i-Ready/seeds/assessments.csv

```diff
@@ -0,0 +1,2 @@
+assessmentIdentifier,assessmentTitle,namespace,assessmentCategoryDescriptor
+Curriculum Associates i-Ready Diagnostic Math and ELA Assess,i-Ready,uri://curriculumassociates.com/Assessment,uri://ed-fi.org/AssessmentCategoryDescriptor#Other
```

Comment:
It might be too late but I think our iready template should match the native integration as much as possible